test/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector.UnitTests/EventLogSessionContextTests.cs

```diff
@@ -70,18 +67,18 @@ public void CreateEventLogContainerShouldCreateNegativeEndIndexIfLogEntriesAreEm
     }
 }
 
-public class DummyEventLogContainer : IEventLogContainer
+public sealed class DummyEventLogContainer : IEventLogContainer
 {
     public DummyEventLogContainer(bool initialize)
     {
         EventLogEntries = new List<EventLogEntry>(10);
-        EventLog eventLog = new("Application");
+        EventLog = new("Application");
```

Comment:
I think there was a "bug" and we were not initializing the right thing (local variable instead of property). Please confirm this was the intent.